resources/views/components/form.antlers.html

```diff
@@ -3,56 +3,61 @@
 <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
 <script>
     axios.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
-    document.getElementById("btn-submit").addEventListener("click", function (e) {
-        e.preventDefault();
-        hideErrorRibbon();
-        const payload = {
-            name: document.getElementById('name').value,
-            email: document.getElementById('email').value,
-            subject: document.getElementById('subject').value,
-            message: document.getElementById('message').value,
-        }
-        const project = document.getElementById('project_id').value;
-        let url = "{{ config:app:contact_api_url }}";
-        if (typeof project !== "undefined" && project !== "") {
-            Object.assign(payload, { project });
-            url = "{{ config:app:contact_project_api_url }}";
-        }
+    const btnSubmit = document.getElementById("btn-submit");
+    if (btnSubmit) {
+        document.getElementById("btn-submit").addEventListener("click", function (e) {
+            e.preventDefault();
+            hideErrorRibbon();
+            const payload = {
+                name: document.getElementById('name').value,
+                email: document.getElementById('email').value,
+                subject: document.getElementById('subject').value,
+                message: document.getElementById('message').value,
+            }
+            const project = document.getElementById('project_id').value;
+            let url = "{{ config:app:contact_api_url }}";
+            let action = "contact_mermaid";
```

Comment:
@saanobhaai I also added **action** condition to be verified by recaptcha, tested in my local.